Victoradukwu/More-Recipes
# Chore: finetune serverside code

#### What does this PR do?
- Refactor backend and client sides
#### Description of Task to be completed?
- Modify the error message of recipe controller
- Include a banner picture on the landing page
- Include error handling for front end
#### How should this be manually tested?
- Start the app with the command `npm run start:dev`
- Go to localhost:8000 to see the landing page
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 - chore/155314935/finetune server side code

#### Screenshots (if appropriate)

#### Questions:
N/A
